dipy/reconst/shm.py

```diff
@@ -997,9 +997,20 @@ def calculate_max_order(n_coeffs):
         Finally, the positive value is chosen between the two options.
         """
 
-        L1 = (-3 + np.sqrt(1 + 8 * n_coeffs)) / 2
-        L2 = (-3 - np.sqrt(1 + 8 * n_coeffs)) / 2
-        return np.int(max([L1, L2]))
+        # L2 is negative for all positive values of n_coeffs, so we don't
```

Comment:
Not sure why this is here. Seems irrelevant to registration. I suppose we need to rebase?

Reply:
I already did a rebase and now when I check, it says that the branch is up-to-date. So, I am a bit lost as to what is going wrong in my Git flow. Trying to figure it out.

Reply:
You were right, I checked again and the earlier rebase was not complete, there were conflicts remaining. I did a rebase again (after pulling the latest changes from the remote origin) and merged the conflicts, manually. This branch should be okay now :)